src/test/regress/sql/pg16.sql

```diff
@@ -138,6 +138,26 @@ SELECT
 FROM
 	(SELECT js::jsonb FROM test_is_json WHERE js IS JSON) foo(js);
 
+-- SYSTEM_USER
+-- Relevant PG commit:
+-- https://github.com/postgres/postgres/commit/0823d061
+
+CREATE TABLE table_name_for_view(id int, val_1 text);
+SELECT create_distributed_table('table_name_for_view', 'id');
+INSERT INTO table_name_for_view VALUES (1, 'test');
+
+-- define a view that uses SYSTEM_USER keyword
+CREATE VIEW prop_view_1 AS
```

Comment:
Does it work like `CURRENT_USER`, see Citus tests, such as `ALTER ROLE CURRENT_USER PASSWORD 'password123' VALID UNTIL 'infinity';`

If it works like `CURRENT_USER`, do we need to add [here](https://github.com/citusdata/citus/blob/371f094b681fd879a4230f24c0a53c48ab3ead59/src/backend/distributed/deparser/citus_ruleutils.c#L1593) and add some tests like that?

Reply:
It doesn't work like CURRENT_USER, it's not part of RoleSpecType-s.